src/libpgmoneta/server.c

```diff
@@ -35,6 +35,7 @@
 #include <security.h>
 #include <server.h>
 #include <utils.h>
+#include <deque.h>
```

Comment:
Sorting

Reply:
I've updated the header includes to be properly sorted.